package.json

```diff
@@ -48,6 +51,7 @@
     "react-text-highlight": "^0.2.0",
     "react-textarea-autosize": "^5.1.0",
     "react-url-query": "^1.1.4",
+    "redux": "^3.7.2",
```

Comment:
What is redux needed for here?

Reply:
@anshumanv , redux was required for the package to be installed

Reply:
Are you sure? 
I'm just looking at the component's repo - https://github.com/kennetpostigo/react-facebook-login-component
I didn't find them stating redux as dependency.
Can you please confirm @stealthanthrax, thanks :smile: